app/models/imports/podcast_rss_import.rb

```diff
@@ -21,7 +21,7 @@ def file_name
   end
 
   def feed_rss
-    config[:feed_rss] ||= http_get(url)
+    @feed_rss ||= http_get(url)
```

Comment:
the issue, or not issue, with this is that we will get updated rss if this is run over a prolonged time
maybe that's fine, but it means an inconsistent src of episodes and data

Reply:
So just thinking this dataflow through...

1. Controller tries to create a new PodcastRssImport
2. [validate_rss](https://github.com/PRX/feeder.prx.org/blob/main/app/models/imports/podcast_rss_import.rb#L66) gets called - which downloads/parses the RSS but _doesn't_ persist any of that
3. Later, in an async PodcastImportJob, we call [import!](https://github.com/PRX/feeder.prx.org/blob/main/app/models/imports/podcast_rss_import.rb#L100)
4. This calls `create_or_update_podcast!`, which again memoizes the RSS.  But also stores the `channel` and `first_entry` to the db.
5. Next during the import, we `create_or_update_episode_imports!` ... this also hits the memoized `feed.entries`, and creates an EpisodeImport with the [entry_hash](https://github.com/PRX/feeder.prx.org/blob/main/app/models/imports/podcast_rss_import.rb#L140) for each of those.
6. EpisodeRssImports run (in their own jobs)... but only reference their already persisted `episode_import.config[:entry]` hash.
7. At the end of each episode import, we call back into [PodcastImport.status_from_episodes!](https://github.com/PRX/feeder.prx.org/pull/1066).  Which is still going to run a `SELECT * ... FOR UPDATE`.  But now, there's much less data in that config field, so it should be quite a bit faster.

So maybe not ideal that we GET the RSS twice now?  But they at least happen pretty close together.  Assuming the worker isn't way behind.